packages/babel-plugin-transform-typescript/package.json

```diff
@@ -13,6 +13,7 @@
     "babel-plugin-syntax-typescript": "7.0.0-alpha.19"
   },
   "devDependencies": {
-    "babel-helper-plugin-test-runner": "7.0.0-alpha.19"
+    "babel-helper-plugin-test-runner": "7.0.0-alpha.19",
+    "typescript": "next"
```

Comment:
Is there a more specific version we can link to make this less likely to break us randomly as things evolve?